app/models/concerns/hyku_addons/solr_document_behavior.rb

```diff
@@ -21,7 +21,7 @@ def attributes
     included do
       class_attribute :registered_attributes
       # Pre-seed registered attributes because they have already been created before this module is included
-      self.registered_attributes = [:identifier, :based_near, :based_near_label, :related_url, :resource_type,
+      self.registered_attributes = [:doi, :identifier, :based_near, :based_near_label, :related_url, :resource_type,
```

Comment:
@cjcolvar, there might be another way of doing this, but i think there is missing `registered_attribute` - `:doi` in the `SolrDocument::DOIBehavior` concerns. When I added it here the Bolognese `doi` method and data value was populated as I would expect.

Reply:
Huh, I'll take a look at this.

Reply:
Good catch.  I think this is the right place to add it for now.